buildtools/install_vs2019_redist.bat

```diff
@@ -0,0 +1,42 @@
+@rem Control variables
+@rem - Redistributable for Visual Studio 2019
+@set vc_redist_install=vc_redist.x64.exe
+
+@rem Determine if running as administrator
+@call :TEST_ADMINISTRATOR
+@if [%errorlevel%]==[10101] goto :END
+
+@rem Install dependencies
+@call :INSTALL_VS2019_REDIST
+@goto END:
+
+:INSTALL_VS2019_REDIST
+@rem Download install file
+@powershell wget https://aka.ms/vs/16/release/%vc_redist_install% -outfile "%TEMP%\%vc_redist_install%"
```

Comment:
"Downloading and installing VS2019 Redist..."

Reply:
Good idea, will do